src/main/webapp/content/scss/themes/theme-dark.scss

```diff
@@ -295,3 +295,14 @@ html {
     font-size: 0.8em;
     color: #a0aabe;
 }
+
+// from https://stackoverflow.com/a/14205976
```

Comment:
This is not necessary

Reply:
I can remove it. I just thought it would be nice to know why this is needed and reference where I got the solution from. I'll reference it in the PR.